### PR TITLE
[AutomaticAssessmentConfiguration] Updates for Xcode13 beta1

### PR DESCRIPTION
**Pull request by tj-devel709**

### CODE REVIEW

File: src/automaticassessmentconfiguration.cs [hunk 1]
```diff
@@ -118,5 +142,41 @@ interface AEAssessmentSessionDelegate {
 
 		[Export ("assessmentSessionDidEnd:")]
 		void DidEnd (AEAssessmentSession session);
+
+		[NoiOS, Mac (12,0), MacCatalyst (15,0)]
+		[Export ("assessmentSessionDidUpdate:")]
+		void DidUpdate (AEAssessmentSession session);
```
Comment (mandel-macaque) [inline src/automaticassessmentconfiguration.cs:148]: No iOS but MacCatalyst? That is weird.

**Review by mandel-macaque — APPROVED** [review]

**Review by rachelkang — COMMENTED** [review]
I'm just wondering - what does this part of the diff signify? How should it be addressed? (from the wiki)

```diff
-API_AVAILABLE(ios(13.4), macos(10.15.4), macCatalyst(14))
-extern NSErrorDomain const AEAssessmentErrorDomain;
+AE_VISIBLE NSErrorDomain const AEAssessmentErrorDomain API_AVAILABLE(ios(13.4), macos(10.15.4), macCatalyst(14));
```

File: src/automaticassessmentconfiguration.cs (hunk 1 above)
Comment (tj-devel709) [reply src/automaticassessmentconfiguration.cs:148]: Oh yeah.. that is interesting

**Comment by tj-devel709** [comment]
> I'm just wondering - what does this part of the diff signify? How should it be addressed? (from the wiki)
> 
> ```diff
> -API_AVAILABLE(ios(13.4), macos(10.15.4), macCatalyst(14))
> -extern NSErrorDomain const AEAssessmentErrorDomain;
> +AE_VISIBLE NSErrorDomain const AEAssessmentErrorDomain API_AVAILABLE(ios(13.4), macos(10.15.4), macCatalyst(14));
> ```

That's a good question. It just looked like noise to me and did not show up on the todos so I just ignored it. Perhaps @mandel-macaque has the answer?

**Comment by mandel-macaque** [comment]
> I'm just wondering - what does this part of the diff signify? How should it be addressed? (from the wiki)
>
>-API_AVAILABLE(ios(13.4), macos(10.15.4), macCatalyst(14))
>-extern NSErrorDomain const AEAssessmentErrorDomain;
>+AE_VISIBLE NSErrorDomain const AEAssessmentErrorDomain API_AVAILABLE(ios(13.4), macos(10.15.4), macCatalyst(14));
>That's a good question. It just looked like noise to me and did not show up on the todos so I just ignored it. Perhaps @mandel-macaque has the answer?

That is noise, someone changing a MACRO, I don't know why they change the macros EVERY YEAR. It is noise.

File: src/automaticassessmentconfiguration.cs (hunk 1 above)
Comment (mandel-macaque) [reply src/automaticassessmentconfiguration.cs:148]: Looking at the docs, is is correct, and that is why I approved, it is just funny that Catalyst was meant to be iOS apps that run on Mac, yet they do things like this. 🍎

**Pull request merged**